lib/middleware.ts

```diff
@@ -81,11 +98,21 @@ var defaultHandlers = [
   // Response size
   (req, res) => ({"response-size": getResponseSize(res)}),
   // Response time (ns)
-  (req, res) => ({"response-time": getResponseTimeNs(req, res)}),
+  (req, res) => ({value: getResponseTimeNs(req, res)}),
   // Status code
   (req, res) => ({"status-code": res.statusCode}),
   // Ip address
   (req) => ({ip: getIp(req)}),
+
+  // Kayvee's reserved fields
+  // Log level
+  (req, res) => ({level: getLogLevel(req, res)}),
+  // Source
+  // -> Gets passed in among `options` during library initialization
+  // Title
+  () => ({title: "response-time"}),
+  // Type
+  () => ({type: "gauge"}),
```

Comment:
I'm not sure this is a requirement, since we will have to make special Heka filters for this logger anyways or both SignalFx and InfluxDB 0.10. However don't have a strong opinion about keeping this either.